templates/base.html

```diff
@@ -79,6 +79,10 @@
                 <nav class="topbar navbar navbar-inverse navbar-static-top" aria-label="{% trans "Main" %}">
                     <div class="container-fluid">
                         <div class="navbar-header">
+							<a class="navbar-brand" href="{% url 'home' %}">{% brand_name %}</a>
```

Comment:
What does this change do? Would you clarify in the discussion here?

Reply:
The brand button had an `anchor` pointing to the home page, but the `anchor` is removed for small devices, and that makes no sense. Users in small devices also expect to brand button to behave as a link to the home page. That is the reason I remove the code below. The code that is shown in green already existed, but the changes in indentation makes the code looks like a new piece of code. I only replaced the `hidden-xs` class with the `visible-xs` class, and move it the **brand** button before the toggle menu.